napi-inl.h

```diff
@@ -613,9 +613,9 @@ inline double Number::DoubleValue() const {
   return result;
 }
 
-// currently experimental guard with version of NAPI_VERSION that it is
+// currently experimental guard with definition of NAPI_EXPERIMENTAL that it is
```

Comment:
```suggestion
// currently experimental guard with version of NAPI_VERSION that it is
```

Reply:
With commit `src: enabling BigInt API with NAPI_EXPERIMENTAL`, BigInt would only be enabled with the definition of `NAPI_EXPERIMENTAL` rather than a very big `NAPI_VERSION`, which is what has been done in `js_native_api.h`.

Reply:
What the comment is trying to say is:

* Currently it is experimental
* When it becomes non-experimental is should be guarded with the NAPI_VERSION in which it became non-experimental (for example it might be version 6).

Maybe it comment should be

// currently experimental, for now guard with definition of NAPI_EXPERIMENTAL.  Once it is no
// longer experimental guard with the NAPI_VERSION in which it is released.